server/database/queries/getPosts.js

```diff
@@ -13,7 +13,7 @@ const getPublicServices = (isDraft, publisherId) => connection.query(`
   from public_service
```

Comment:
in ``` getEvents ``` you should select id, title just you don't need to get category 
in ``` getPublicServices ``` you should select id, title just you don't need to get primary_tag


@AbdallahAmmar96  i think that is your fault

Reply:
We need it in the front end @knanahassouna1. When clicking on the post, we push to the history the post url which includes the category/primary_tag. Make sense?